### PR TITLE
Update assignmentCreationWithRecordForm.html

### PR DESCRIPTION
**Pull request by 121910319020**
hi bharat give me an update of the code.

### CODE REVIEW

File: assignmentCreationWithRecordForm/assignmentCreationWithRecordForm.html [hunk 1]
```diff
@@ -10,4 +10,5 @@
     <lightning-input label="Status" value={status} onchange={handleInputChange}></lightning-input>
     <lightning-button label="Create Record" onclick={handleCreateRecord}></lightning-button>
     <lightning-button label="happy"></lightning-button>
+    <lightning-button label="happy">We are the best mowa</lightning-button>
```
Comment (saibharath2805) [inline assignmentCreationWithRecordForm/assignmentCreationWithRecordForm.html:13]: We are not best mowa we are ok

**Review by saibharath2805 — CHANGES_REQUESTED** [review]
need enhancements

File: assignmentCreationWithRecordForm/assignmentCreationWithRecordForm.html (hunk 1 above)
Comment (121910319020) [inline assignmentCreationWithRecordForm/assignmentCreationWithRecordForm.html:13]: ok will work on it

Comment (saibharath2805) [reply assignmentCreationWithRecordForm/assignmentCreationWithRecordForm.html:13]: ok work

**Pull request merged**